fenix/app/src/main/java/org/mozilla/fenix/debugdrawer/ui/DebugDrawerFAB.kt

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugdrawer.ui
```

Comment:
with this package addition, the thought is to have something like:

`debugdrawer/ui` for all of the Compose code
`debugdrawer/redux` for the store/state/middlewares
`debugdrawer/feature_name/` for any of the functionality (tab manager, secret settings, crash viewer, etc.)

Reply:
perhaps we should use debugsettings as the overall feature name? Then Debug Drawer would be part of the UI package, wdyt?

Reply:
Both of these make sense to me